test/NVQPP/custom_pass.cpp

```diff
@@ -0,0 +1,32 @@
+/*******************************************************************************
+ * Copyright (c) 2022 - 2023 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+// clang-format off
+// RUN: nvq++ --enable-mlir --opt-plugin %cudaq_lib_dir/CustomPassPlugin.so --opt-pass 'func.func(cudaq-custom-pass)'  %s -o %t && %t | FileCheck %s
```

Comment:
Q: Is this plugin already being built?

Reply:
Yea, it is built as [part of our unit tests](https://github.com/NVIDIA/cuda-quantum/tree/main/unittests/plugin).

Reply:
Can you double check that the cmake dependency is correct? Thanks.